lib/utils/index.js

```diff
@@ -260,7 +260,8 @@ class Utils {
         }
       }
     }
-
+    parentFolder = this.isFileNameValid(parentFolder) ? '' : parentFolder;
```

Comment:
Instead of this, a more general way of doing this would be adding the following on Line 255:
`parentFolder = path.dirname(folder).split(path.sep).pop();`

Reply:
@AlaaSayed794 could you make this change, the rest looks good to me. Again thanks for the PR.

Reply:
@swrdfish the thing is that src_folder could be path as an array of folders (suites) which is the default.
The PR is to handle if the user passes an array of files.

but the default would be an array of folders where `parentFolder = path.dirname(folder).split(path.sep).pop();` would always go up one level.

I ran it on the same tests anyway and it didn't work well with the second one being passed some folders.